net-nds/389-ds-base/389-ds-base-1.4.2.15.ebuild

```diff
@@ -1,39 +1,40 @@
 # Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
+EAPI=7
 
-WANT_AUTOMAKE="1.13"
+WANT_AUTOMAKE="1.16"
 
-inherit user eutils multilib flag-o-matic autotools
-
-DESCRIPTION="389 Directory Server (core librares and daemons )"
-HOMEPAGE="http://www.port389.org/"
-SRC_URI="http://www.port389.org/sources/${P}.tar.bz2"
+inherit eutils multilib flag-o-matic autotools systemd db-use
 
+DESCRIPTION="389 Directory Server (core librares and daemons)"
+HOMEPAGE="https://directory.fedoraproject.org/"
+MY_P="${PN}-${PN}-${PV}"
+SRC_URI="https://pagure.io/389-ds-base/archive/${PN}-${PV}/${MY_P}.tar.gz"
```

Comment:
is there a reason why you took these over the tarballs located on https://directory.fedoraproject.org/docs/389ds/development/source.html ?

Reply:
No reason, but the build instructions I followed were on the pagure.io git page, so that was where I got the releases from. Happy to change.